subcommand/server-acl-init/command_test.go

```diff
@@ -1008,6 +1112,71 @@ func TestRun_AlreadyBootstrapped(t *testing.T) {
 	}, consulAPICalls)
 }
 
+// Test if there is a provided bootstrap we skip bootstrapping of the servers
+// and continue on to the next step.
+func TestRun_SkipBootstrapping_WhenBootstrapTokenIsProvided(t *testing.T) {
+	t.Parallel()
+	require := require.New(t)
+	k8s := fake.NewSimpleClientset()
+
+	bootToken := "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+	tokenFile, fileCleanup := writeTempFile(t, bootToken)
+	defer fileCleanup()
+
+	type APICall struct {
+		Method string
+		Path   string
+	}
+	var consulAPICalls []APICall
+
+	// Start the Consul server.
+	consulServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Record all the API calls made.
+		consulAPICalls = append(consulAPICalls, APICall{
+			Method: r.Method,
+			Path:   r.URL.Path,
+		})
+		switch r.URL.Path {
+		case "/v1/agent/self":
+			fmt.Fprintln(w, `{"Config": {"Datacenter": "dc1"}}`)
+		default:
+			// Send an empty JSON response with code 200 to all calls.
+			fmt.Fprintln(w, "{}")
+		}
+	}))
+	defer consulServer.Close()
+
+	// Create the Server Pods.
```

Comment:
This gets a server url, it doesn't create pods.

Reply:
Good catch! (:shakes_fist: at git rebase)